2.4.0.M2/arm64/alpine/Dockerfile

```diff
@@ -0,0 +1,93 @@
+# openhab image
+#
+# ------------------------------------------------------------------------------
+#               NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+#                       PLEASE DO NOT EDIT IT DIRECTLY.
+# ------------------------------------------------------------------------------
+#
+FROM multiarch/alpine:arm64-v3.7
+
+# Set download urls
+ENV 	    JAVA_URL="https://www.azul.com/downloads/zulu/zdk-8-ga-linux_aarch32hf.tar.gz" 	    OPENHAB_URL="https://openhab.jfrog.io/openhab/libs-milestone-local/org/openhab/distro/openhab/2.4.0.M2/openhab-2.4.0.M2.tar.gz" 	    OPENHAB_VERSION="2.4.0.M2"
```

Comment:
There was also a commit to nicely layout these ENV lines which seems to be missing from your branch.

Reply:
Fixed